### DISCUSSION TITLE
cs50 IDE Codespace not setting up

### DISCUSSION THREAD

**Discussion by rfaley** [community/community#30837] [Codespaces]
Hello,

After selecting 'Log in via Github' on http://code.cs50.io/ the next page becomes an infinite loop between 'Setting up your Codespace' and 'Codespace is disconnected'. CS50 admins were not able to troubleshoot this, please assist.

Codespace link: https://rfaley-code50-112112395-gjprvq5pw6w2p456.github.dev/

Thanks!

Ryan

**Comment by osortega** [comment]
Hello @rfaley! 👋 
Thanks for reporting! 

We are looking into this, in the meantime could you help us verifying if you are able to connect through VS Code desktop?
Also could you give it try connecting to the codespace from a private window? There's a possibility that something in the browser is messing with the requests.

**Reply by osortega** [reply]
Looks like something went wrong with the codespace for a bit until eventually it self-recovered.
You should be able to connect to the codespace fine now.

We are still looking into the issue to make sure it doesn't happen again.

**Comment by rfaley** [comment]
Hey Osvaldo,

Thanks for the quick response! It doesn't appear to be working through the
browser still but does appear to be working through the desktop.

Ryan

**Comment by osortega** [comment]
@rfaley Did you try from a private window disabling all browser extensions?
